src/com/aca/homework/week4/snake/Snake.java

```diff
@@ -0,0 +1,64 @@
+package com.aca.homework.week4.snake;
+
+public class Snake {
```

Comment:
Maybe better to name this class BoardPixel or SnakePart, because obviously, it is not a Snake